libcxxabi/test/guard_threaded_test.pass.cpp

```diff
@@ -26,15 +26,21 @@
 
 using namespace __cxxabiv1;
 
+#ifndef __ve__
```

Comment:
We very much want to avoid platform specific behavior like this in our tests. This doesn't scale since we need to support 10s of platforms which often have small variations like this. We do have some platform-specific behavior in a few places, but we try to limit it where it is truly necessary.

We could perhaps do something with `std::thread::hardware_concurrency()` -- what does that return on your platform?

Reply:
Thank you.  `std::thread::hardware_concurrency()` returns 8.  We have 8 cores per a VE.

Reply:
@ldionne -san, do you havee any idea to implement this kind of limitation in common methods?

We can use `std::thread_:hardware_concurrency()` for at least `constexpr int ConcurrentRunsPerTest = 8;`...  Thanks anyway.

Reply:
Sorry for the delay. I do not think there's a Standard way to do this, since `hardware_concurrency` is not directly tied to the number of supported threads, and in the case of VE both differ.

If we use 

```
constexpr int ThreadsPerTest = 8;
constexpr int ConcurrentRunsPerTest = 8;
```

then we would have 64 threads exactly, right? Would that pass on VE?

Reply:
Gentle ping.